compiler/src/iree/compiler/Dialect/Stream/Transforms/test/unify_encoding_for_globals.mlir

```diff
@@ -0,0 +1,200 @@
+// RUN: iree-opt --split-input-file --pass-pipeline='builtin.module(iree-stream-unify-encoding-for-globals)' %s | FileCheck %s
+
```

Comment:
IIUC, the test currently does not yet test the case where the source global doesn't have an inline initializer and we need to trace through stores.